app/models/performance_statistics.rb

```diff
@@ -6,38 +6,9 @@ def initialize(year)
   end
 
   def candidate_query
-    year_clause = year ? "AND f.recruitment_cycle_year = #{year}" : ''
```

Comment:
I might be missing something, but this refactor doesn't look related to the PR?

Reply:
Good spot - did a few too many branch switches and comitted the wrong thing.